packages/dgt-components/lib/components/authentication/webid.component.ts

```diff
@@ -149,6 +152,9 @@ export class WebIdComponent extends RxLitElement {
           background-color: var(--colors-primary-light);
           border: 2px solid var(--colors-primary-light);
         }
+        .button-icon-right {
+          display: none;
+        }
```

Comment:
Have you tried unsetting `display: none` when using the authenticate component? I don't think you will be able to any styling on this div unless you export it as a part